package.sh

```diff
@@ -196,7 +181,7 @@ dpkg-deb -Zgzip --root-owner-group --build $STG_DIR/deb/$pkg_dir
 
 generate_rpm_package rpm
 generate_rpm_package suse
-generate_rpm_package mariner
+generate_rpm_package stunnel
```

Comment:
Can you clarify why this needs to be separate? Is it being used explicitly somewhere? How does "generate_rpm_package rpm" differ from "generate_rpm_package stunnel"?

Reply:
yes, I thought you said you don't need a separate package for stunnel now. The package will be either rpm (for non-suse distros) or suse (for suse).

Reply:
Yes, I removed the extra packages but forgot to cleanup this part of the script.